src/utils/signaling.js

```diff
@@ -1299,6 +1305,33 @@ Signaling.Standalone.prototype.processEvent = function(data) {
 	}
 }
 
+Signaling.Standalone.prototype.processDialOutEvent = function(data) {
+	if (data.dialout.callid) {
+		store.dispatch('processDialOutAnswer', { callid: data.dialout.callid })
+	} else if (data.dialout.error) {
+		console.debug(data.dialout.error)
+	}
+}
+
+Signaling.Standalone.prototype.processTransientEvent = function(data) {
+	switch (data.transient.type) {
+	case 'set':
+		if (data.transient.key.startsWith('callstatus_')) {
+			store.dispatch('processTransientCallStatus', { value: data.transient.value })
+		}
+		break
+	case 'remove':
+		// ignore event
```

Comment:
we saw that a status `cleared` has type `remove`. Shall we call `dispatch('deletePhoneState')` from here?

Reply:
All status changes are sent with type `set`. The `remove` is a couple of seconds after the status was set to `cleared` / `rejected` to cleanup the entries from the transient data. Otherwise the object would just grow indefinitely.

Also you might want to handle the `initial` type which will contain the status values for all currently existing calls when a (web) user joins later.

Reply:
Is there a received signaling message to indicate muted/ on hold status or will it be in `initial` type?  We have only seen this data 
![image](https://github.com/nextcloud/spreed/assets/84044328/757cf64c-471e-495f-80af-0a7403be415f)